web/packages/teleterm/src/ui/Vnet/VnetConnectionItem.tsx

```diff
@@ -106,6 +116,12 @@ const VnetConnectionItemBase = forwardRef<
     showDiagWarningIndicator,
     isDiagSupported,
   } = useVnetContext();
+  const { close: closeConnectionsPanel } = useConnectionsContext();
+  const rootClusterUri = useStoreSelector(
+    'workspacesService',
+    useCallback(state => state.rootClusterUri, [])
+  );
+  const isUserInWorkspace = !!rootClusterUri;
```

Comment:
Assigning this to a variable just so that the ternary in the return value is easier to read.